CMakeLists.txt

```diff
@@ -143,6 +143,15 @@ list(APPEND CMAKE_PREFIX_PATH ${CMAKE_INSTALL_PREFIX} ${CMAKE_INSTALL_PREFIX}/ll
 option(ENABLE_HIP_WORKAROUNDS Off)
 set(MIOPEN_INSTALL_CXX_HEADERS Off CACHE BOOL "Install MIOpen's C++ header interface")
 
+# Build flags for backward compatibility of MIOPEN Compiler flags
+set( MIOPEN_COMPILER_FLAG_VERSION V1 CACHE STRING
+        "Which version of Compiler Flags to use" )
+
+if(MIOPEN_COMPILER_FLAG_VERSION STREQUAL "V2" )
+  set(MIOPEN_COMPILER_FLAGS_V2 1)
+  message( STATUS "MIOPEN COMPILER FLAGS VERSION 2 using getROCmInstallPath()" )
+endif()
```

Comment:
Please rename this to MIOPEN_USE_COMPILER_FLAGS_V2 and make it boolean (On/Off). I highly recommend re-using used to define MIOPEN_USE_COMGR:
https://github.com/ROCm/MIOpen/blob/5c7617c9241272cec6b6ae64fab18a59182f1b46/CMakeLists.txt#L251-L252
Please move this definition as close as possible to the line where it is first used (line 495).

Reply:
[resolved]
Continued at https://github.com/ROCm/MIOpen/pull/2694#pullrequestreview-1873288561.